src/main/webapp/js/instructorFeedbackResults.es6

```diff
@@ -140,11 +140,6 @@ function filterResults(rawSearchText) {
     }
 }
 
-function updateResultsFilter() {
-    $('input[id=filterTextForDownload]').val($('#results-search-box').val());
-    filterResults($('#results-search-box').val());
```

Comment:
Oops sorry for not being clear. What I meant was the function named `filterResults`.